appyters/Gene_Expression_by_Tissue/Gene_Expression_by_Tissue.ipynb

```diff
@@ -117,10 +176,14 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "%%appyter code_exec\n",
+    "\n",
+    "{% if gtex_gene.value %}\n",
```

Comment:
For all of the new conditionals, you'll need to use `.raw_value`
`value` contains the sanitized field value which would be the string "True or "False" when coerced to a boolean this will always be true. `.raw_value` on the other hand has the actual boolean but is unsafe to use outside of conditionals since the user could in-fact provide an invalid string over API or something.

Reply:
Or did I fix this hmmm.. maybe can you just verify that the conditional works as false?

Reply:
I stand corrected, looks like I fixed that :man_shrugging: LGTM